### ISSUE TITLE
can not find module 'xxxxx' after packaged

### ISSUE THREAD

**Issue by Harbon** [electron/electron#5071]
* Electron version: 0.37.2
* Operating system: win10
every thing works perfectly when i run 'electron main.js', after I packaged, console show error can not find module 'xxxx', every module i wrote in package.json can not be found. I did 'console.log(module.paths)', my app node_modules 'path actually be included, how can i fix this?

**Comment by kevinsawicki** [comment]
Are you sure your `node_modules` directory is present in your packaged app and that it includes all the modules your app needs?

**Comment by Harbon** [comment]
yes,I checked the resources/app directory, all modules I needs is exist. After doing  some experiments, I found that if you push the node_modules's path of you develop directory to  module.paths it works perfectly, or (I mean the default node_modules in resource/app/node_modules) it fail. I use electron-packager ,so I guess something wrong when electron-packager copy the node_modules from my develop directory. If I download the template and copy my node_module to resources manually it works fine, so I think it's the issue of electron-packager not the electron itself. Thank you!!!!
